E/EDFlib/build_tarballs.jl

```diff
@@ -12,7 +12,7 @@ sources = [
 script = raw"""
 cd ${WORKSPACE}/srcdir/edflib
 mkdir ${libdir}
-${CC} edflib.c -shared -fPIC -o ${libdir}/libedflib.${dlext}
+${CC} edflib.c -shared -fPIC -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -o ${libdir}/libedflib.${dlext}
```

Comment:
Does `_LARGEFILE64_SOURCE` apply to all platforms?

Reply:
I think so
```
 - Macro: _LARGEFILE64_SOURCE
     If you define this macro an additional set of functions is made
     available which enables 32 bit systems to use files of sizes beyond
     the usual limit of 2GB.  This interface is not available if the
     system does not support files that large.  On systems where the
     natural file size limit is greater than 2GB (i.e., on 64 bit
     systems) the new functions are identical to the replaced functions.

     The new functionality is made available by a new set of types and
     functions which replace the existing ones.  The names of these new
     objects contain `64' to indicate the intention, e.g., `off_t' vs.
     `off64_t' and `fseeko' vs. `fseeko64'.
```

Reply:
I found that online so it must be true